### DISCUSSION TITLE
内容管理总览 | FixIt

### DISCUSSION THREAD

**Discussion by giscus** [hugo-fixit/FixIt#208] [General]
# 内容管理总览 | FixIt

了解如何在 FixIt 主题中快速，直观地创建和组织内容。

https://fixit.lruihao.cn/zh-cn/docs/content-management/introduction/

**Comment by Meidanlong** [comment]
您好，想请教两个问题：

文中提到，“保持博客文章存放在 content/posts 目录”

1、如果不在这个目录下的博客，则不会渲染“创作时间”、“目录”、“底部评论”等功能？能否统一渲染？
2、content/posts目录下有多个文件夹，如“前段文件夹”和“后端文件夹”，目前也是依据具体md的文件名区分路径，能否自动映射路径。如“localhost:1212/posts/backend/nacos/”

感谢

**Reply by Lruihao** [reply]
1. 如果不在这个文件夹，在文章front matter 加上 type: posts 即可
2. 可以自定义映射路径，比如你分前后端文件夹，可以放在 /content/frontend/ 和 /content/backend/，然后加上 type: posts, 如果需要自定义路径详见 https://gohugo.io/content-management/urls/#permalinks

**Reply by Meidanlong** [reply]
感谢

**Comment by weiran4** [comment]
您好！麻烦想问一下，您文章左边的slider合集索引目录是怎么弄出来的，右边目录toc似乎必须post才有，doc就没有，我按照您源代码_index设定了以后也没有，toml都是默认的

感谢！

**Reply by Lruihao** [reply]
1. 首先检查一下配置了是否有合集的分类设置。

  ```toml
  [taxonomies]
   _merge = "shallow"

  [params.page]
  collectionNavigation = true
  ```

  有的话，文章前面像分类标签一样指定合集 collections 即可。

2. 目录只有在类型为posts的页面显示，如果你的文章目录不是posts，例如doc，那你可以在doc/_index.md中手动批量设置：

  ```
  ---
  title: Documentation
  cascade:
    params:
      type: posts
  ---
  ```

**Reply by weiran4** [reply]
感谢回复！

我按照您说的改了，确实文章末尾有了一个收录于XXX合集，并且可以跳转下一篇的窗口出来了，但是左边还是没有导览的
<img width="233" height="419" alt="Snipaste_2025-12-22_19-54-30" src="https://github.com/user-attachments/assets/05b422d4-720c-4606-823c-3e188b091dc1" />
我的doc\下的_index复制粘贴你的，type是post，doc\文件夹下的_index也是collection: outline，然后这个文件夹内每个文章的collection也是一类的，toml也加了，还是不行。。。

麻烦您了

**Reply by Lruihao** [reply]
多加一个，collectionList=true。

类型是 posts  

合集是 collections

**Reply by weiran4** [reply]
感谢感谢，终于解决了，问chat半天也没搞定
